src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -12,13 +13,16 @@
 
 import filter.expression.QualifierType;
 import javafx.scene.control.ContextMenu;
-import javafx.scene.control.Label;
 import javafx.scene.control.MenuItem;
 import javafx.scene.input.KeyCodeCombination;
 import javafx.scene.input.KeyEvent;
 import javafx.scene.layout.HBox;
 import javafx.scene.layout.Priority;
+import javafx.scene.text.Text;
+import javafx.scene.paint.Color;
+
 
+import javafx.scene.text.TextFlow;
```

Comment:
Not sure if your IDE settings create this but 1 blank line should be enough. Can also your IDE's automatic imports organization for these stuffs.